cpp/src/io/comp/gpuinflate.cu

```diff
@@ -1222,4 +1230,40 @@ void gpu_copy_uncompressed_blocks(device_span<device_span<uint8_t const> const>
   }
 }
 
+sorted_codec_parameters sort_tasks(device_span<device_span<uint8_t const> const> inputs,
```

Comment:
Optional: Since this function returns newly allocated device memory, shouldn't it also accept an `mr`. Though I can understand it's always gonna be the default one